core/flyout.js

```diff
@@ -1131,7 +1201,10 @@ Blockly.Flyout.prototype.placeNewBlock_ = function(originBlock) {
   }
   // Figure out where the original block is on the screen, relative to the upper
   // left corner of the main workspace.
-  var xyOld = targetWorkspace.getSvgXY(svgRootOld);
+  var xyOld = Blockly.utils.getInjectionDivXY_(svgRootOld);
```

Comment:
if (targetWorkspace.isMutator) {
  var xyOld = this.workspace_.getSvgXY(...);
} else {
  var xyOld = Blockly.utils.getInjectionDivXY_(...);
}